### DISCUSSION TITLE
Is it possible to have dummy text when using custom script?

### DISCUSSION THREAD

**Discussion by ilhamfu** [davatorium/rofi#1916] [Q&A] [answered]
Like the title said. Context i want to make youtube playlist player, and i want to display the name of the playlist and the playlist url, but i don't want the url to be searchable. is it possible?

**Comment by DaveDavenport** [comment, marked as answer]
You can add a [non-searchable](https://davatorium.github.io/rofi/current/rofi-script.5/#parsing-row-options) url for each row,  but it won't be displayed. 
Anything displayed is searched.

**Reply by ilhamfu** [reply]
that's unfortunate actually, but it is what it is

**Reply by DaveDavenport** [reply]
Added an option to git that should make this possible.
